string.js

```diff
@@ -0,0 +1,103 @@
+//  1. `console.log` the first character of a string.
+ let string1= 'garcia'
+ console.log(string1[0])
+//
+//  2. `console.log` the length of a string.
+console.log(string1.length)
+//
+//  3. `console.log` the last character of any string.
+ console.log(string1[string1.length-1])
+
+//  4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+ let str3 = "juan"
+ console.log(str3.toUpperCase(str3[str3.length-1])) // capitalizes all characters.
+console.log('------------------')
+ let name= 'juan'
+ let capitalized = name.slice(name.length) + name[name.length-1].toUpperCase()
+ console.log(name.slice(0,3) + capitalized)
+
+
+//  5. Create a drEvil code block that will take a single number variable, and log the '<variableAmount> dollars',
+//  your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
+let amount = 5
+if (amount < 1000000){
+console.log(amount + ' dollars')
+} else if (amount >= 1000000){
+  console.log(amount + "(pinky)")
+}
+
+//   6. Create a `verbing` code block. It should take a single string variable. If its length is at least 3,
+// it should add 'ing' to its end, unless it already ends in 'ing',
+// in which case it should add 'ly' instead. If the string length is less than 3, it should leave it unchanged.
+//  For example:
+//
+
+let o = 'walk'
+if (o.length >=3 && o.endsWith('ing')){
+  console.log(o + 'ly')
+} else if (o.length>=3){
+  console.log(o + 'ing')
+}
+
+//  7. Create a `withoutLast` code block that takes a single string variable, and returns a copy of the string without the last word.
+let statement = "I need coffee now"
+let withoutLast1 = statement.split(" ", 3);
+console.log(withoutLast1)
+
+let statement2 =  "I hate you"
```

Comment:
Another instance of hard coding. This code wouldn't work if I change "I hate you" to "I hate CheetoFace".